Git_GitHub/ The_Unix_Shell.md

```diff
@@ -203,9 +201,16 @@ Output
 ```
 This overwrites what is in quotes.txt with the content of the test1.txt, so it has to be used with caution.
 ### **Move file to another directory**
-Lets ay that we want to move the file data.doc from the chapter_1 folder to the oen above directory which is the project_1. We again use ***mv*** but with different arguments.
+Lets say that we want to move the file (animals.csv) from the chapter_1 folder to one directory (up) or directory above which is (project_1), and then it's subdirectory (chapter_2). We again use ***mv*** but with different arguments.
 
-To move in the nesting :  /Users/mary/project_1/chapter_1
+**[DW]** Need example, something like this?
+**[MT]** I give thsi example to demonstrate that we can copy the file (animals.csv)  in another directory/subdirectory (../chapter_2/animals.csv)  and show that it doesnt exist in the firts dicrectory any longer.
+
+
+To move in across the directory structure, let's say we are at /Users/mary/project_1/chapter_1, we want to move animals.csv there to /Users/mary/project_1/chapter_2:
+
+**[DW]** the animals.csv (in chapter_2) is not necessary, or do you mean to explain another topic?[/DW] 
+**[MT]** i wanted to show that we move files (not necessary documents) from one directory to othe directory with difefrent path
 
 ```bash
 mv animals.csv ../chapter_2/animals.csv   
```

Comment:
```suggestion
mv animals.csv ../chapter_2/.   
```
Don't need to put animals twice. You can just put the . there to indicate you want to keep the same name as the file.